### ISSUE TITLE
Enable Multiple Types

### ISSUE THREAD

**Issue by terrellt** [ActiveTriples/ActiveTriples#125]
What do we see as the API for this? Multiple type configurations?
```ruby
class Resource < ActiveTriples::Resource
  configure :type => RDF::URI("http://athing.com")
  configure :type => RDF::URI("http://anotherthing.com")
end
```

**Comment by jcoyne** [comment]
That looks reasonable to me

**Comment by no-reply** [comment]
The problem with multiple types is that the (ruby) Class of the resource is tied to type--and not in a terribly clear or well documented way.  I think we need to resolve this before (or in the process of) making changes to current type configuration.

**Comment by terrellt** [comment]
@no-reply That's true. Can we add this functionality without breaking current behavior, via casting on the first defined type? PCDM is currently blocked here, and I want to explore options.

**Comment by no-reply** [comment]
:+1: if you can get something working that either doesn't alter existing behavior or improves clarity, I'm in favor.  Just use care; here be dragons.

**Comment by terrellt** [comment]
:+1: Will do. Pretty sure I spawned some of those dragons, so it's my burden to bear.

**Comment by no-reply** [comment]
This was closed by #126 #128 #129 & #130.

Thanks for keeping it simple @terrellt! :P
